### ISSUE TITLE
How to uninstall?

### ISSUE THREAD

**Issue by NightMachinery** [jlund/streisand-discussions#139]

**Comment by nopdotcom** [comment]
Uninstall the server? I don't think there is a good one. What you could do is disable services, and adjust the `ufw` firewall if you wanted to reuse the machine. I recommend a scratch reinstall, though. Many cloud providers have "reinstall the OS" actions. Which one are you using?

**Comment by NightMachinery** [comment]
I have installed some other stuff, so I don’t want to do an OS reinstall.
How can I disable services?

**Comment by nopdotcom** [comment]
Most things are controlled by systemd. Look through the output of `systemctl list-units`, and then 

```
systemctl disable wg-quick@wg0.service
systemctl disable streisand-wireguard
```

You can use `systemctl mask` instead of `disable` if you want to make doubly-sure they don't run.

Look through the results of `netstat -lp` for what programs are running. As a last resort, use `ufw reject portnumber` to block stuff.

Beware there's going to be a bunch of configuration left over that's not in a standard state. For example, `sslh` is running on port 443, redirecting connections to daemons on different ports. See `/etc/sslh.cfg`.

**Comment by NightMachinery** [comment]
I just want to free the ram. Tor in particular eats a lot of ram.

**Comment by nopdotcom** [comment]
Yeah, just `systemctl disable tor` should do it. Lemme know if that works.
